app/models/manageiq/providers/ibm_power_hmc/infra_manager/lpar.rb

```diff
@@ -36,11 +36,94 @@ def raw_rename(new_name)
   def make_template(template_name)
     $ibm_power_hmc_log.info("#{self.class}##{__method__} ems_ref #{ems_ref} template_name #{template_name}")
     ext_management_system.with_provider_connection do |connection|
-      host_uuid = connection.lpar(ems_ref).sys_uuid
-      connection.capture_lpar(ems_ref, host_uuid, template_name)
+      connection.capture_lpar(ems_ref, host.ems_ref, template_name)
```

Comment:
This looks unrelated but a nice change, maybe in a separate PR?

Reply:
I realized we had this unnecessary call to the SDK gem, I can move it out